### PR TITLE
Adding buildscript for symfony/finder

### PR DESCRIPTION
**Pull request by vathsalashetty25**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@kumarvikas2020 please review

File: s/symfony__finder/symfony_finder_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+if [ -d "$PACKAGE_NAME" ] ; then
+  rm -rf $PACKAGE_NAME
+  echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"
+fi
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+    exit 0
```
Comment (kumarvikas2020) [inline s/symfony__finder/symfony_finder_ubi_8.5.sh:39]: please use exit 1 for any error condition.

**Comment by vathsalashetty25** [comment]
Updated the script with required changes.

**Comment by seth-priya** [comment]
@kumarvikas2020 please confirm changes

**Review by kumarvikas2020 — APPROVED** [review]

**Pull request merged**